### ISSUE TITLE
is this still working?

### ISSUE THREAD

**Issue by CrimeNow** [Jire/Charlatano#194]
is this hack still working guys? safe to usage?

**Comment by LoveIsDeath** [comment]
Yeah have not been banned...

**Comment by Is82jM** [comment]
I played with it yesterday on my main and not banned.

**Comment by CrimeNow** [comment]
I try this hax but esp didnt work, Aimbot same. Just RCS worked but it was buggy.

**Comment by Is82jM** [comment]
@CrimeNow I played 3 games in MM right now with GlowESP(skeleton and BOX doesn't work for me).

**Comment by CrimeNow** [comment]
thanks man after installing win10 ill try again :)

**Comment by arsh1129** [comment]
ex-aimtux user?? ;-P

**Comment by CrimeNow** [comment]
Yep :D Lwss aimtux fork is working. But cant say its ud or not fukin delayed bans :D This hack is minimal i liked it :D And no fps drops like in linux :D

**Comment by SquareHeadd** [comment]
esp used to work for me on earlier versions of charlatano. but for some reason its not working atm. 
Can someone (someone with a workin esp) please send me their "Charlatano 1.0.3" folder...

**Comment by CrimeNow** [comment]
I installed Win 10. Esp is working now. When i was using 8.1 Esp and aim bot did not work. Now box esp doesnt work but maybe its only in fullscreen windowed .
